ui/pages/confirmations/confirmation/alerts/setAlerts.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useMemo } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  Alert,
+  clearAlerts,
+  updateAlerts,
+} from '../../../../ducks/confirm-alerts/confirm-alerts';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+const setAlerts = (alertOwnerId: string) => {
```

Comment:
For consistency and to be explicit, could we call this something like `useTemplateConfirmationAlerts`?